### PR TITLE
TELCODOCS#2091: Release note for editing the over-provisioning ratio

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s):
4.18

Issue:
[TELCODOCS-2091](https://issues.redhat.com/browse/TELCODOCS-2091)

Link to docs preview:
- [Storage](https://86316--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-18-release-notes.html#ocp-release-notes-storage_release-notes) (New features and enhancements)

QE review:
- [x] QE has approved this change.

Note to merge reviewers: SME and Peer reviews for this content done in this PR - https://github.com/openshift/openshift-docs/pull/85368. I noticed that I had not created a feature branch for creating the previous PR. Hence, created this PR for merging.

### CODE REVIEW

**Review by michaelryanpeter — COMMENTED** [review]
These suggestions are minor, but I still think it would be better to fix them before we merge. Especially since this is for a future release.

/remove-label merge-review-needed
/remove-label merge-review-in-progress

File: release_notes/ocp-4-18-release-notes.adoc [hunk 1]
```diff
@@ -468,6 +468,11 @@ Updated boot images has been promoted to GA for Amazon Web Services (AWS) cluste
 [id="ocp-release-notes-storage_{context}"]
 === Storage
 
+[id="ocp-4-17-storage-editing-overprovisioning-ratio"]
```
[the remark this answers is not in the document]
Comment (sr1kar99) [reply release_notes/ocp-4-18-release-notes.adoc:471]: Updated as per the suggestion.

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM. Sorry for the delay, your other PR sent me down a rabbit hole of fixing unrelated issues that were causing checks to fail 😅

**Pull request merged**